pkgs/applications/misc/volnoti/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub
+, pkgconfig, dbus, gdk_pixbuf, glib, libX11, gtk2
+, dbus_glib, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  name = "volnoti-${version}";
+  version = "git-2019-09-26";
+
+  src = fetchFromGitHub {
+    owner = "davidbrazdil";
+    repo = "volnoti";
+    rev = "4af7c8e54ecc499097121909f02ecb42a8a60d24";
+    sha256 = "155lb7w563dkdkdn4752hl0zjhgnq3j4cvs9z98nb25k1xpmpki7";
+  };
+
+  patches = [ ./dbus-headers-fix.patch ];
+
+  preConfigure = ''
+   ./prepare.sh
```

Comment:
This script just runs `autoreconf`, which `autoreconfHook` already takes care of so can remove the `preConfigure` field completely.